.dockerignore

```diff
@@ -0,0 +1 @@
+docker-compose
```

Comment:
why this?
whats the reason that there is a Dockerffile in the main root anyway?

Reply:
That builds the bleeding edge dev docker.  It's the unstable one used for kevin's repo